packages/backend-core/src/cache/writethrough.ts

```diff
@@ -4,7 +4,7 @@ import { logWarn } from "../logging"
 import { Database, Document, LockName, LockType } from "@budibase/types"
 import * as locks from "../redis/redlockImpl"
 
-const DEFAULT_WRITE_RATE_MS = 10000
+const DEFAULT_WRITE_RATE_MS = 5000
```

Comment:
@mike12345567 During some tests in local I've noticed a race condition related with the usage of cache. I've decreased this number and now I don't have that problems but I think it is a weak solution. What's the purpose of this usage? What problems could be derived from this change? Is this change acceptable?

in line 42 we can find this:
```javascript
const cache = await getCache()
  const key = doc._id
  let cacheItem: CacheItem | undefined
  if (key) {
    cacheItem = await cache.get(makeCacheKey(db, key))
  }
  const updateDb = !cacheItem || cacheItem.lastWrite < Date.now() - writeRateMs
```

Quotas for users are not just a sum but a complete recalculation based on persisted values; if I make a change in DB and 8 seconds later I make another one, it will take the value from cache and no update in DB will be performed. This change reduce the changes of this kind of problems but it doesn't avoid them.

Should I modify the code to use `writeRateMs = 0` instead changer the DEFAULT_WRITE_RATE_MS? What impact could have this kind of changes in performance?